hazelcast/src/main/java/com/hazelcast/internal/nio/NodeIOService.java

```diff
@@ -73,74 +71,35 @@
     public NodeIOService(Node node, NodeEngineImpl nodeEngine) {
         this.node = node;
         this.nodeEngine = nodeEngine;
-        restApiConfig = initRestApiConfig(node.getProperties(), node.getConfig());
-        memcacheProtocolConfig = initMemcacheProtocolConfig(node.getProperties(), node.getConfig());
+        restApiConfig = initRestApiConfig(node.getConfig());
+        memcacheProtocolConfig = initMemcacheProtocolConfig(node.getConfig());
     }
 
     /**
-     * Initializes {@link RestApiConfig} if not provided based on legacy group properties. Also checks (fails fast)
-     * if both the {@link RestApiConfig} and system properties are used.
+     * Initializes {@link RestApiConfig} if not provided based on legacy group properties.
```

Comment:
Javadoc needs update, no props  involved anymore.

Reply:
Would you mind if I remove it entirely? As for me method body speaks for itself. Thanks

Reply:
In that particular case I agree. Go for it.

Reply:
Done. I believe it is ready for merge? Thanks